stripe-connect/src/main/java/com/stripe/android/connectsdk/EmbeddedComponentManager.kt

```diff
@@ -1,7 +1,170 @@
 package com.stripe.android.connectsdk
 
+import android.os.Parcelable
+import androidx.activity.ComponentActivity
+import androidx.annotation.ColorInt
 import androidx.annotation.RestrictTo
+import kotlinx.parcelize.Parcelize
 
 @PrivateBetaConnectSDK
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-class EmbeddedComponentManager internal constructor()
+class EmbeddedComponentManager internal constructor() {
+
+    // TODO MXMOBILE-2760 - replace with actual implementation and remove the @Suppress annotations
+    constructor(
+        @Suppress("UNUSED_PARAMETER") activity: ComponentActivity,
+        @Suppress("UNUSED_PARAMETER") configuration: Configuration,
+        @Suppress("UNUSED_PARAMETER") fetchClientSecret: FetchClientSecretCallback,
```

Comment:
Are we avoiding coroutines in the public API for Java compatibility or something? I see we already have coroutine libraries as dependencies. What are our plans for reducing friction for Kotlin codebases?

```suggestion
        @Suppress("UNUSED_PARAMETER") fetchClientSecret: suspend () -> String,
```

Reply:
Let's have this discussion as part of API review - this API isn't yet final. You're right on though, suspend functions in the public API are basically unusable from Java which is why I went with this.

Kotlin functional interfaces do make this very idiomatic to use from Kotlin, although admittedly not quite as nice: https://kotlinlang.org/docs/fun-interfaces.html I could also see us providing a suspend helper function to make this nicer too.